matchers/caseInsensitive/todomatcher.go

```diff
@@ -0,0 +1,39 @@
+package caseInsensitive
+
+import (
+	"regexp"
+
+	"github.com/preslavmihaylov/todocheck/matchers"
+)
+
+type TodoMatcherCaseInsensitive struct {
+	matcher matchers.TodoMatcher
+}
+
+func NewTodoMatcherCaseInsensitive(matcher matchers.TodoMatcher) *TodoMatcherCaseInsensitive {
```

Comment:
Should be `NewTodoMatcher` as the usage would be `case_insensitive.NewTodoMatcher`. As it is right now, usage would be very verbose